libcontainer/mount/mount.go

```diff
@@ -21,3 +26,20 @@ func Mounted(mountpoint string) (bool, error) {
 	}
 	return false, nil
 }
+
+// FSType returns the file-system type of a mountpoint.
+//
+// ErrNotMounted is returned if the mountpoint refers to a path
+// that is not mounted.
+func FSType(mountpoint string) (string, error) {
```

Comment:
Wouldn't it be simpler to do an `statfs` and check whether `f_type == PROC_SUPER_MAGIC`?

Reply:
done